modules/appeals_api/spec/requests/v0/appeals_controller_spec.rb

```diff
@@ -121,72 +107,45 @@
-                    'X-VA-User' => 'adhoc.test.user'
-                  }
-                )
-            expect(response).to have_http_status(:bad_request)
-          end
+          get(path, params: nil, headers: { 'X-Consumer-Username' => 'TestConsumer', 'X-VA-User' => 'adhoc.test.user' })
+          expect(response).to have_http_status(:bad_request)
         end
       end
     end
 
     context 'with a not found response' do
       it 'returns a 404 and logs an info level message' do
         VCR.use_cassette('caseflow/not_found') do
-          with_openid_auth(oauth_scopes) do |auth_header|
-            get endpoint,
-                params: nil,
-                headers: auth_header.merge(
-                  {
-                    'X-VA-SSN' => '111223333',
-                    'X-Consumer-Username' => 'TestConsumer',
-                    'X-VA-User' => 'adhoc.test.user'
-                  }
-                )
-            expect(response).to have_http_status(:not_found)
-            expect(response).to match_response_schema('errors')
-          end
+          get(
+            path,
+            params: nil,
+            headers: {
+              'X-VA-SSN' => '111223333',
+              'X-Consumer-Username' => 'TestConsumer',
+              'X-VA-User' => 'adhoc.test.user'
+            }
+          )
+          expect(response).to have_http_status(:not_found)
+          expect(response).to match_response_schema('errors')
         end
       end
     end
 
     context 'with a response where "aod" is null instead of a boolean' do
       it 'returns a successful response' do
         VCR.use_cassette('caseflow/appeals_null_aod') do
-          with_openid_auth(oauth_scopes) do |auth_header|
-            get endpoint,
-                params: nil,
-                headers: auth_header.merge(
-                  {
-                    'X-VA-SSN' => '111223333',
-                    'X-Consumer-Username' => 'TestConsumer',
-                    'X-VA-User' => 'adhoc.test.user'
-                  }
-                )
-          end
+          get(
+            path,
+            params: nil,
+            headers: {
+              'X-VA-SSN' => '111223333',
+              'X-Consumer-Username' => 'TestConsumer',
+              'X-VA-User' => 'adhoc.test.user'
+            }
+          )
           expect(response).to have_http_status(:ok)
         end
       end
     end
-
-    if opts[:oauth_scopes].present?
-      context 'with oauth' do
-        it_behaves_like 'an endpoint with OpenID auth', scopes: opts[:oauth_scopes] do
-          def make_request(auth_header)
-            VCR.use_cassette('caseflow/appeals') do
-              get('/services/appeals/v1/appeals', headers: auth_header.merge(
-                { 'X-VA-SSN' => '111223333', 'X-Consumer-Username' => 'TestConsumer', 'X-VA-User' => 'adhoc.test.user' }
-              ))
-            end
-          end
-        end
-      end
-    end
```

Comment:
Since this was converted from a shared examples file to an Appeals Status API v0 spec, the OAuth examples are no longer included here (v0 does not use OAuth).